src/test/java/net/minecraftforge/debug/world/RaidEnumTest.java

```diff
@@ -0,0 +1,19 @@
+package net.minecraftforge.debug.world;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.world.raid.Raid;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod("raid_enum_test")
+public class RaidEnumTest 
+{
+	private static final boolean ENABLE = false;
+	
+	public RaidEnumTest()
+	{
+		if (ENABLE)
```

Comment:
Formatting issues.

Reply:
i mean, this is a test mod, but i will fix it i guess once i get back on my pc